Werehouse/Werehouse.py

```diff
@@ -0,0 +1,50 @@
+helmet = 0
+rifle = 0
+medkit = 0
+
+print('Hello in our werehouse')
+while True:
+    print("We have: " + str(helmet) + " helmets, " + str(rifle) + " rifles, and " + str(medkit) + " medkits"  )
+    Temp = input("Type 1 to add or 2 for out item: ")
+    if Temp == "1":
+
+        Temp1 = input("Type item to add : ")
+        if Temp1 == "helmet":
+            Temp2 = input("how many helmets would You like to add :")
+            helmet += int(Temp2)

```

Comment:
what will happen if I will provide `asdf` as the input? I think we will see the exception and program will end - please handle such case telling the user that only positive numbers are accepted

Reply:
which of asdf?[ ](https://pl.wikipedia.org/wiki/ASDF) ;)

thanks, I just figured out, that if You type a negative number, there is negative count of items - I have to fix it!